src/transformers/configuration_xlnet.py

```diff
@@ -163,6 +163,10 @@ def __init__(
         self.n_layer = n_layer
         self.n_head = n_head
         assert d_model % n_head == 0
+        if "d_head" in kwargs:
+            assert kwargs["d_head"] == d_model // n_head, (
+                f"`d_head` ({kwargs['d_head']}) should be " f"equal to `d_model // n_head` ({d_model // n_head})"
```

Comment:
Formatting was weird

```suggestion
                f"`d_head` ({kwargs['d_head']}) should be equal to `d_model // n_head` ({d_model // n_head})"
```